tests/py3-ignores.txt

```diff
@@ -27,8 +25,6 @@ tests/test_engine.py
 tests/test_http_cookies.py
 tests/test_logformatter.py
 tests/test_mail.py
-tests/test_pipeline_files.py
-tests/test_pipeline_images.py
```

Comment:
Some context: we decided to keep Headers pure bytes (both for keys and values) for 3 reasons:

1. this is what Twisted is doing;
2. it is easiest to implement;
3. different headers can be in different encodings - e.g. cookies are usually in UTF-8, while other headers are often latin-1.

This is not a final decision, but something we did to unblock further porting. The biggest downside is that code which works in Python 2.x silently fails in Python 3.x - an example is `headers.get(''Referrer")` which should be changed to `headers.get(b"Referrer")`.